gammapy/stats/counts_statistic.py

```diff
@@ -61,6 +61,48 @@ def p_value(self):
         """
         return 0.5 * chi2.sf(self.ts, 1)
 
+    def __str__(self):
+        str_ = "\t{:32}: {{n_on:.2f}} \n".format("Total counts")
+        str_ += "\t{:32}: {{background:.2f}}\n".format("Total background counts")
+        str_ += "\t{:32}: {{excess:.2f}}\n".format("Total excess counts")
+        str_ += "\t{:32}: {{significance:.2f}}\n".format("Total significance")
+        str_ += "\t{:32}: {{p_value:.3f}}\n".format("p - value")
+        str_ += "\t{:32}: {{n_bins:.0f}}\n".format("Total number of bins")
+        info = self.info_dict()
+        info["n_bins"] = np.array(self.n_on).size
+        str_ = str_.format(**info)
+
+        return str_.expandtabs(tabsize=2)
+
+    def info_dict(self, summed=True, axis=None):
```

Comment:
Actually I think we could just remove the `summed` option. User can achieve the same, like:
```python3
stat.sum().info_dict()
```
Instead of:
```python3
stat.info_dict(summed=True)
```
I think there is no clear advantage for the latter case. Not that we have the pattern of "hierarchical" function calls also elsewhere, e.g. `map.smooth(1).plot()` or `map.sum_over_axes().plot()`.